code/addons/docs/package.json

```diff
@@ -51,6 +51,11 @@
       "import": "./dist/preset.mjs",
       "types": "./dist/preset.d.ts"
     },
+    "./mdx-react-shim": {
+      "require": "./dist/shims/mdx-react-shim.js",
+      "import": "./dist/shims/mdx-react-shim.mjs",
+      "types": "./dist/shims/mdx-react-shim.d.ts"
+    },
```

Comment:
We do this, so we don't also have to add a dependency on `mdx-react` for the user.